lib/pure/collections/sequtils.nim

```diff
@@ -202,6 +202,41 @@ proc deduplicate*[T](s: openArray[T], isSorted: bool = false): seq[T] =
       for itm in items(s):
         if not result.contains(itm): result.add(itm)
 
+proc minIndex*[T](s: openArray[T]): int {.since: (1, 1).} =
+  ## Returns the index of the minimum value of `s`.
```

Comment:
* should make it clear what happens in case of duplicate such indexes (`the` index is incorrect / imprecise); eg: `In case of duplicates, the smallest such index is returned`
* arguable, but IMO returning `-1` for s.len == 0 may make more sense